cram/cram_external.c

```diff
@@ -82,6 +82,14 @@ void cram_container_set_num_blocks(cram_container *c, int32_t num_blocks) {
     c->num_blocks = num_blocks;
 }
 
+int32_t cram_container_get_num_records(cram_container *c) {
+    return c->num_records;
+}
+
+int32_t cram_container_get_num_bases(cram_container *c) {
```

Comment:
Good spot.  I naively assumed it'd never be more than 32-bit because that would be an insanely large block (I think CRAM can't cope with that anyway), but CRAM 4 could. :)